src/views/Languages/index.jsx

```diff
@@ -96,22 +140,25 @@ export default class Languages extends Component {
               <Button
                 className={classes.button}
                 size="large"
-                aria-owns={anchorEl ? 'fade-menu' : null}
-                aria-haspopup="true"
-                onClick={this.handleClick}>
-                {lang || 'Pick languages'}
-                <MenuDownIcon />
+                onClick={this.handleDrawerToggle}>
+                Skills
               </Button>
-              <Menu id="fade-menu" anchorEl={anchorEl} open={Boolean(anchorEl)}>
-                {options.map(item => (
-                  <MenuItem key={item} onClick={this.handleClose} id={item}>
-                    {item}
-                  </MenuItem>
-                ))}
-              </Menu>
             </Grid>
           </Grid>
         </AppBar>
+        <Drawer
+          variant="temporary"
+          anchor="right"
+          open={drawerOpen}
+          onClose={this.handleDrawerToggle}
+          classes={{
+            paper: classes.drawerPaper,
+          }}
+          ModalProps={{
+            keepMounted: true,
+          }}>
+          {drawer}
+        </Drawer>
```

Comment:
Picking a skill shows an empty table. Is this on purpose?

![screen shot 2018-07-30 at 10 10 28 am](https://user-images.githubusercontent.com/3766511/43402509-e6e31ada-93e0-11e8-99dc-f491b635c80c.png)

Reply:
yes it is supposed to be handled by other PR #163